src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -16,10 +16,18 @@ internal class Emitter
             private const int MaxLoggerMessageDefineArguments = 6;
             private const int DefaultStringBuilderCapacity = 1024;
 
+            private static readonly string s_generatedTypeSummary =
+                "<summary> " +
+                "The implementation of this type is subject to change without further notice, " +
+                "do not use it directly in your code. " +
```

Comment:
I think I would slightly adjust the wording here, following the wording we use in our API docs ([example](https://github.com/dotnet/runtime/blob/635cfb3409fcce4257e35ba0cf3d385fdee3a6ad/src/libraries/System.Linq.Expressions/src/System/Runtime/CompilerServices/Closure.cs#L10)).

```suggestion
                "This API supports the logging infrastructure and is not intended to be used directly from your code. " +
                "It is subject to change in the future. " +
```